src/runtime/interop35.cs

```diff
@@ -3,7 +3,7 @@
 
 
 
-#if (PYTHON35)
+#if (PYTHON35 || PYTHON36)
```

Comment:
This is a leftover from trying to reuse the file for Python 3.6. It is no longer needed. I'll remove this change. Thank you.